src/cassandra/batch.rs

```diff
@@ -43,50 +41,6 @@ impl Protected<*mut _Batch> for Batch {
     }
 }
 
-/// Custom payloads not fully supported yet
```

Comment:
Please can you update CHANGELOG to reflect your changes in this PR?

Reply:
Yes.